lib/gap_intelligence/client/requestable.rb

```diff
@@ -8,21 +8,21 @@ module Requestable
     # @yield [req] The Faraday request
     def perform_request(method, path, options = {}, &block)
       record_class = options.delete(:record_class)
+      response_processor = options.delete(:response_processor)
```

Comment:
@shavkat-samatov, as `response_processor` is not used, I guess we can revert all these changes because nothing changed in the logic.